boot/zephyr/flash_map_extended.c

```diff
@@ -55,9 +55,9 @@ int flash_device_base(uint8_t fd_id, uintptr_t *ret)
  */
 int flash_area_id_from_image_slot(int slot)
 {
-    static const int area_id_tab[] = {FLASH_AREA_IMAGE_PRIMARY,
-                                      FLASH_AREA_IMAGE_SECONDARY,
-                                      FLASH_AREA_IMAGE_SCRATCH};
+    const int area_id_tab[] = {FLASH_AREA_IMAGE_PRIMARY,
```

Comment:
Following change will decrees flash footprint by 24 B.
```suggestion
#if (MCUBOOT_IMAGE_NUMBER == 1)
    static
#endif
    const int area_id_tab[] = {FLASH_AREA_IMAGE_PRIMARY,
```

Reply:
Thanks, I've added the conditional directives to the code.